src/main/resources/templates/layouts/main.html

```diff
@@ -17,8 +17,11 @@
 <body id="app">
 
 <nav id="page-navbar" class="navbar">
-    <a href="" data-th-href="@{/}" class="page-navbar__item page-navbar__item-selected">
-        <ion-icon name="home"></ion-icon>
+    <a data-th-href="@{/}" class="page-navbar__item" th:classappend="${#request.requestURI == '/probes' ? 'page-navbar__item-selected' : ''}">
+        <ion-icon name="home" data-th-src="@{/svg/home.svg}"></ion-icon>
```

Comment:
this is definitively not the solution.

Reply:
Hmm, why is it "definitively not" the solution? I guess it's because I've hardcoded the path, right?

Reply:
Sure, I'd revert this.

Reply:
Are you able to see the icons on `probes/results` after this commit: https://github.com/jenkins-infra/plugin-health-scoring/pull/102/commits/4fafd29f4e586193ccc1dc9f9ff4bd4e5e76d7be? I'm still not seeing them after I pulled the latest changes, am I missing something?